keyboards/mechlovin/hannah65/rev1/haus/readme.md

```diff
@@ -0,0 +1,15 @@
+# Haus
+
+![Haus](https://i.imgur.com/sQ2QWAZh.jpg)
+
+A Haus PCB, Rev.1
+
+* Keyboard Maintainer: [Hand Engineering](https://www.reddit.com/r/MechanicalKeyboards/comments/hzcavx/haus_by_hand_engineering/)
+* Hardware Supported: Haus PCB rev.1, STM32F303.
+* Hardware Availability: Haus Keyboard.
+
+Make example for this keyboard (after setting up your build environment):
+
+    make mechlovin/hannah65/rev1/haus:default
```

Comment:
```suggestion
    make mechlovin/hannah65/rev1/haus:default

Flashing example for this keyboard:

    make mechlovin/hannah65/rev1/haus:flash
    
```

Reply:
Can you also add a line on how to put the board into bootloader mode? (if there is a switch, how to use it, etc)